packages/calcite-components/src/utils/dom.ts

```diff
@@ -204,10 +204,19 @@ export function queryElementRoots<T extends Element = Element>(
  * @param {string} selector The selector.
  * @returns {Element} The targeted element.
  */
+export function closestElementCrossShadowBoundary<TagName extends keyof HTMLElementTagNameMap>(
+  element: Element,
+  selector: TagName,
+): HTMLElementTagNameMap[TagName] | null;
+export function closestElementCrossShadowBoundary<T extends Element = Element>(
+  element: Element,
+  selector: string,
+): T | null;
 export function closestElementCrossShadowBoundary<T extends Element = Element>(
   element: Element,
   selector: string,
 ): T | null {
+  document.createElement;
```

Comment:
This seems unrelated. Is it intentional?

Reply:
Indeed - I used it for testing, but forgot to remove - thanks for catching